hathor/builder/cli_builder.py

```diff
@@ -143,19 +143,15 @@ def create_manager(self, reactor: PosixReactorBase, args: Namespace) -> HathorMa
 
         pubsub = PubSubManager(reactor)
 
-        event_manager: Optional[EventManager] = None
         if args.x_enable_event_queue:
             self.event_ws_factory = EventWebsocketFactory(reactor, event_storage)
-            event_manager = EventManager(
-                event_storage=event_storage,
-                event_ws_factory=self.event_ws_factory,
-                pubsub=pubsub,
-                reactor=reactor,
-                emit_load_events=args.x_emit_load_events
-            )
-        else:
-            self.check_or_raise(not args.x_emit_load_events, '--x-emit-load-events cannot be used without '
-                                                             '--x-enable-event-queue')
+
+        event_manager = EventManager(
+            event_storage=event_storage,
+            event_ws_factory=self.event_ws_factory,
+            pubsub=pubsub,
+            reactor=reactor
+        )
```

Comment:
Exactly. The `EventManager` is still used to retrieve the last event queue feature state (either enabled or disabled) from when the full node was previously initialized, but it can only be used for the actual event queue mechanism if `_enable_event_queue` is `True`